### DISCUSSION TITLE
C# SDK - Catalog Items and List Items queries return no Attributes

### DISCUSSION THREAD

**Discussion by kevnhowe** [amzn/selling-partner-api-models#3759] [General]
Testing both CatalogItems and ListingItems APIs and neither return Attribute values for Get methods.
I have checked the ProductTypeDefinition API for the availability of the selected listings product types, and I can see that the Type is available.
I am getting items back, using all IncludedData enums in the request, including "attributes"
Each item returned has an empty attribute set.
Are there additional steps I need to take to return all available attribute values for a given listing?

**Comment by stefnats** [comment]
I'm not using c# but Java. In an old version, there was an issue parsing the attributes data, which is now fixed. 
I think the c# client is somehow having the same error. Can you verify that the actual raw json response has no attribute data?

**Comment by kevnhowe** [comment]
correct. the json response has no attribute data.
